manifest-generation/bosh-lite-stubs/property-overrides.yml

```diff
@@ -381,6 +381,8 @@ property_overrides:
       network: tcp
       address: 127.0.0.1:7777
     log_level: debug
+    volman:
+      driver_paths: /var/vcap/data/voldrivers
```

Comment:
I don't think we need to add this default here as we have it in the job spec. We should be able to leave this property out of this stub entirely.